src/relion/dbmodel/modeltables.py

```diff
@@ -55,15 +41,15 @@ def __getitem__(self, key):
         return self._tab[key]
 
     @staticmethod
-    def _make_list(elem, default=[]):
+    def _make_list(elem, default=...):
+        # default value fallacy
         if elem is None:
-            return default
+            return [] if default is Ellipsis else default
         elif isinstance(elem, list):
             return elem
         return [elem]
```

Comment:
This function used a list as a default argument, meaning [every invocation gets the _same_ list](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments):
```python
>>> def test(x=[]):
>>>     return x
>>> x = test()
>>> y = test()
>>> x.append(1)
>>> x
[1]
>>> y
[1]
```

How this didn't cause issues for you I don't know :thinking:

(There's probably a nicer way to resolve this)

Reply:
I keep doing this 🤦‍♂️. It'll sink in eventually. Could change the default of `default` to `None` and then set `default=[]` where it is called inside of `__init__`?

Reply:
Done